tests/back/api/test_runner.py

```diff
@@ -144,12 +144,14 @@ def test_exec_runs(client: TestClient, run_id: int):
 
     response = client.get(f"/api/v1/run/{run_id}")
     data = response.json()
-    assert data["train_metrics"] is not None
-    assert data["validation_metrics"] is not None
-    assert data["test_metrics"] is not None
+    assert type(data["train_metrics"]) is dict
+    assert type(data["validation_metrics"]) is dict
+    assert type(data["test_metrics"]) is dict
```

Comment:
Use `isinstance` instead:

```suggestion
    assert isinstance(data["train_metrics"], dict)
    assert isinstance(data["validation_metrics"], dict)
    assert isinstance(data["test_metrics"], dict)
```